onnxruntime/test/python/onnx_backend_test_series.py

```diff
@@ -110,6 +110,11 @@ def create_backend_test(testname=None):
         # if c2.supports_device('NGRAPH'):
         #    current_failing_tests = current_failing_tests + ('|^test_operator_repeat_dim_overflow_cpu.*',)
 
+        # Failing for Nuphar.
+        if c2.supports_device('NUPHAR'):
```

Comment:
Would be good to say why it fails and whether that could/should be fixed.

Reply:
Sure

---
In reply to: [312430383](https://github.com/microsoft/onnxruntime/pull/881#discussion_r312430383) [](ancestors = 312430383)